BlueprintUI/Sources/Internal/Extensions/FloatingPoint+ScaleRounding.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+extension FloatingPoint {
```

Comment:
(copied this from BlueprintUICommonControls so that it can be internal to this module)

Reply:
just noticed that there's also a `ScaleRoundingTests` that I didn't copy over; anyone have opinions on whether I should move it (rather than copy it) into BlueprintUI and make it public?

Reply:
I'm fine with either, though it's nice to expose this in BlueprintUI so others can use it when writing their own elements, IMO.

Reply:
I've floated this idea before, but I think we want to avoid polluting `FloatingPoint` publicly. We could wrap the behavior up in a separate type to avoid that, but it's outside the scope of your change.

For now, I think copying this is fine. I've opened #27 to track the followup.

Reply:
@watt sounds good, thanks!